sunkit_image/asda.py

```diff
@@ -11,410 +11,379 @@
 
 from sunkit_image.utils import calculate_gamma, points_in_poly, reform2d, remove_duplicate
 
-__all__ = ["Asda", "Lamb_Oseen"]
+__all__ = [
+    "gen_vel",
+    "gamma_values",
+    "center_edge",
+    "vortex_property",
+    "get_grid",
+    "get_vtheta",
+    "get_vradial",
+    "get_vxvy",
+]
+
+
+def gen_vel(vx, vy, i, j, r=3):

```

Comment:
```suggestion
def generate_velocity_field(vx, vy, i, j, r=3):
```
I think its time we improved some of the names as well.

Reply:
I also think i, j and r should be looked at for a rename.

Reply:
I will try to come up with suggestions.